dr_ensenso_node/src/ensenso.cpp

```diff
@@ -32,6 +35,46 @@
 
 #include <memory>
 
+namespace {
+
+NxLibItem selectNxItmCalibration(std::string const & camera, std::unique_ptr<dr::Ensenso> & ensenso_camera) {
```

Comment:
No need to take a reference to a `unique_ptr` here. A regular reference or maybe even const reference would be more flexible. Same for `getCameraParams` below.

You can always get a plain reference from a `unique_ptr` for free, but not the other way around.